<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2005</width>
    <height>966</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#ffffff;&quot;&gt;Load Video&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>790</y>
      <width>1280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1280</width>
      <height>50</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_play">
    <property name="geometry">
     <rect>
      <x>127</x>
      <y>860</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Play or Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_frame_index">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>850</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string>(0/0)</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_scene_start">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>860</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>[</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_scene_end">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>860</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>]</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_save_all_scenes">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>860</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Save All Scenes</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>10</y>
      <width>691</width>
      <height>891</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_scene_init">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>860</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Init</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_video_load">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>860</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Load Video</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_scene_progress_bar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>740</y>
      <width>1280</width>
      <height>40</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2005</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
